index.html

```diff
@@ -3116,7 +3116,7 @@ <h3>Receiving and parsing content</h3>
               </li>
             </ol>
             <p class="note">
-              Using the encoder from [[!ENCODING]], it is only possible
+              Using the encoder from [[ENCODING]], it is only possible
               to encode as UTF-8, unless you do the encoding manually
               and use the <code>"opaque"</code>
```

Comment:
```suggestion
              and use the `"opaque"`
```

Reply:
I'd like to unify this in a separate PR. :)

Reply:
sure, sounds good.